ml4h/tensor_generators.py

```diff
@@ -828,14 +875,31 @@ def test_train_valid_tensor_generators(
         paths=test_paths, num_workers=num_train_workers, cache_size=0, weights=weights,
         keep_paths=keep_paths or keep_paths_test, mixup_alpha=0, name='test_worker', siamese=siamese, augment=False,
     )
-    if wrap_with_tf_dataset:
+
+    do_augmentation = bool(rotation_factor or zoom_factor or translation_factor)
+    logging.info(f'doing_augmentation {do_augmentation}')
```

Comment:
It's already in the log at the top with all of the arguments so this not needed, but if you want to keep the info statement here I would add the actual values of the augmentation arguments.